website/docs/r/hpc_cache.html.markdown

```diff
@@ -172,6 +180,14 @@ A `dns` block contains the following:
 
 * `search_domain` - (Optional) The DNS search domain for the HPC Cache.
 
+---
+
+An `identity` block supports the following:
+
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this HPC Cache. Possible value is `UserAssigned`.
+
+* `identity_ids` - (Optional) Specifies a list of User Assigned Managed Identity IDs to be assigned to this HPC Cache.
```

Comment:
there's common phrasing for these now, can we pull these from another resource

Reply:
Have updated to use the common phrasing